lib/business_time/currency.rb

```diff
@@ -0,0 +1,24 @@
+module BusinessTime::Currency
+  # @return [Array] the list of currencies
+  # @param [String+]
+  def args(*currency)
+    return nil if currency.length.zero?
```

Comment:
```ruby
return nil if currency.empty?
```

The fact, that the code came from somebody else, does not automatically mean that it is perfect :)